src/components/leads/SimulateEmailCampaignDialog.tsx

```
@@ -0,0 +1,252 @@
+
+    return (
+      <div className="flex-1 p-6 overflow-y-auto">
+        <div className="mb-4">
+          <h3 className="text-sm font-medium text-gray-700 mb-1">Subject:</h3>
+          <p className="text-base font-medium text-gray-900">{email.subject}</p>
+        </div>
+        <div>
+          <h3 className="text-sm font-medium text-gray-700 mb-2">Body:</h3>
+          <div 
+            className="prose prose-sm max-w-none text-gray-900"
+            dangerouslySetInnerHTML={{ __html: email.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)